MaxText/layers.py

```diff
@@ -1176,9 +1200,10 @@ def __call__(self,
                 max_decode_length)
 
     y = LayerNorm(dtype=cfg.dtype, name='decoder_norm', kernel_axes = ('embed',))(y)
-    y = nn.Dropout(
-        rate=cfg.dropout_rate, broadcast_dims=(-2,))(
-            y, deterministic=deterministic)
+    #No Dropout in Llama
```

Comment:
Instead of commenting out dropout, could we do
if cfg.dropout_rate is not None:
   y = nn.Dropout(...)

And configure llama model with dropout rate is None?

Reply:
Same as other droput layers.

Reply:
Brought back all the Dropout removals as suggested by Rafi since

by default dropout_rate=0

https://github.com/google/maxtext/blob/main/MaxText/configs/base.yml#L54
https://github.com/google/maxtext/blob/main/MaxText/layers.py#L1070